packages/xo-server/src/api/xosan.js

```diff
@@ -367,6 +367,7 @@ async function glusterCmd(glusterEndpoint, cmd, ignoreError = false) {
   }
   if (result.exit === 0) {
     const cliOut = result.parsed.cliOutput
+    // TODO: check if cliOut.opRet and cliOut.opErrno are strings or numbers because 'parseXml' function has changed
```

Comment:
If we're unsure if they are numbers or strings, we can simply cast them as number and compare them to `0`.